tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -202,10 +202,9 @@ protected void importOSM() {
                     measureRouting(hopper, new QuerySettings("routing_edge", count / 20, isCH, isLM).
                             withInstructions().edgeBased());
                     // unfortunately alt routes are so slow that we cannot really afford many iterations
-                    // actually it is just too slow so we skip it for now
-//                    measureRouting(hopper, new QuerySettings("routing_edge_alt", count / 1000, isCH, isLM).
-//                            edgeBased().alternative()
-//                    );
+                    measureRouting(hopper, new QuerySettings("routing_edge_alt", count / 1000, isCH, isLM).
+                            edgeBased().alternative()
```

Comment:
Currently we use `count=5000`, so `count / 1000` means only five routes will be measured. Could be still useful, or could fluctuate too much. Let's see. Otherwise we could try something like `count / 500`.